testing/src/main/java/org/oppia/android/testing/ApiUtils.kt

```diff
@@ -1,4 +1,4 @@
-package org.oppia.android.data.backends
+package org.oppia.android.testing
```

Comment:
Hmm maybe something like ``testing/network`` would be a clearer place for this--WDYT?

Reply:
Makes sense -- I wasn't sure if this would be limited to the network services or if it could be used elsewhere but that change be made if we ever end up doing that.

Done